unittests/allplatformstests.py

```diff
@@ -3713,10 +3713,26 @@ def test_wrap_git(self):
                   patch_directory = wrap_git_builddef
                   revision = master
                 '''.format(upstream_uri)))
-            self.init(srcdir)
+            out = self.init(srcdir)
             self.build()
             self.run_tests()
 
+            # Make sure the warning does not occur on the first init.
+            out_of_date_warning = r'revision may be out of date'
+            self.assertNotRegex(out, out_of_date_warning)
```

Comment:
```suggestion
            self.assertNotIn(out, out_of_date_warning)
```

Reply:
And below, assertRegex -> assertIn

Reply:
Changed `self.assertNotRegex(out, out_of_date_warning)` to ` self.assertNotIn(out_of_date_warning, out)`